prov/efa/src/rxr/rxr_pkt_type_base.c

```diff
@@ -311,11 +311,11 @@ int rxr_ep_flush_queued_blocking_copy_to_hmem(struct rxr_ep *ep)
  * 			On failure, return libfabric error code
  */
 static inline
-int rxr_pkt_copy_data_to_hmem(struct rxr_ep *ep,
-			      struct rxr_pkt_entry *pkt_entry,
-			      char *data,
-			      size_t data_size,
-			      size_t data_offset)
+int rxr_pkt_queued_copy_data_to_hmem(struct rxr_ep *ep,
```

Comment:
`queued_copy` or `copy_queued` ?

Reply:
should be `queued_copy`. Updated the commit log